email_service_client/client/types.py

```diff
@@ -11,3 +15,24 @@ class EmailIdentity(TypedDict):
     identity: str
     identity_type: str
     mail_from_subdomain: str
+
+
+class OutgoingEmailRequestPayload(TypedDict):
+    from_name: str
+    from_address: str
+    recipients: list[str]
+    cc: list[str]
+    bcc: list[str]
+    reply_to: list[str]
+    subject: str
+    body: str
+    html: str
+    amp: str
+    source_uuid: UUID
+    attachments: list[AttachmentDetails]
```

Comment:
Replace `list` with `typing.List` to fix the tests.

We should use `AttachmentDetails` of type TypedDict instead of namedtuple, this ensures typesafety

Reply:
But `AttachmentDetails` is supposed to be a tuple!

Reply:
@sushant-plivo @amalshaji-plivo ^

Reply:
Oh cool

Reply:
```python
class AttachmentDetails(NamedTuple):
    filename: str
    content: str
    mimetype: str
```

You can use `NamedTuple` class to preserve typings